### ISSUE TITLE
Can not import pdf files

### ISSUE THREAD

**Issue by alexpattyn** [flxzt/rnote#79]
# Specs
Rnote version: 0.3.1

# Current behavior
When I open or import a pdf with rnote it just crashes.

Below is the cli output when I launch rnote and import a pdf. 

```
$ flatpak run com.github.flxzt.rnote

(rnote:2): Gtk-CRITICAL **: 16:35:41.552: gtk_widget_set_parent: assertion '_gtk_widget_get_parent (widget) == NULL' failed

(rnote:2): Gtk-CRITICAL **: 16:35:41.553: gtk_widget_set_parent: assertion '_gtk_widget_get_parent (widget) == NULL' failed

(rnote:2): Gtk-CRITICAL **: 16:35:41.597: gtk_widget_set_parent: assertion '_gtk_widget_get_parent (widget) == NULL' failed

(rnote:2): Gtk-CRITICAL **: 16:35:41.597: gtk_widget_set_parent: assertion '_gtk_widget_get_parent (widget) == NULL' failed

(rnote:2): Gtk-CRITICAL **: 16:35:41.628: gtk_widget_set_parent: assertion '_gtk_widget_get_parent (widget) == NULL' failed

(rnote:2): GStreamer-WARNING **: 16:35:41.680: loop detected in the graph of bin 'marker_pipeline'!!

(rnote:2): Gdk-CRITICAL **: 16:35:53.507: gdk_memory_texture_new_subtexture: assertion 'x < 0 || x >= GDK_TEXTURE (source)->width' failed

(rnote:2): Gdk-CRITICAL **: 16:35:53.507: gdk_texture_get_width: assertion 'GDK_IS_TEXTURE (texture)' failed

(rnote:2): Gdk-CRITICAL **: 16:35:53.507: gdk_texture_get_height: assertion 'GDK_IS_TEXTURE (texture)' failed

```

# Expected behavior
Previously (v0.2.5 or so) it was able to open pdfs without issues.

**Comment by flxzt** [comment]
could you provide the failing pdf if is not sensitive to you? I'd like to replicate it on my machine

**Comment by alexpattyn** [comment]
Just upgraded to v0.3.2 of rnote and the issue seems resolved :man_shrugging: Should've made sure I was fully updated before reporting.

**Comment by flxzt** [comment]
No worries, you opened the issue before I pushed v0.3.2. And I am not sure it actually solved the issue, so if it comes up again please reopen so we can investigate it further.

**Comment by alexpattyn** [comment]
Alright I'll keep an eye out for it again. When I tried to reproduce it earlier I didn't have any issues.

**Comment by alexpattyn** [comment]
Just wanted to reopen as I was experiencing this issue again. See the attached pdf.

[DNN-Part I.pdf](https://github.com/flxzt/rnote/files/8028014/DNN-Part.I.pdf)

**Comment by flxzt** [comment]
I can replicate it with your PDF. The crashes seem to be dependent on the actual accumulated pixel resolution of all pages the imported PDF. I'll investigate further later

**Comment by alexpattyn** [comment]
Thanks for looking into it. Let me know if you needed anything or for me to test something. That pdf was created by converting a pptx with onlyoffice - perhaps onlyoffice does something weird when converting certain power points?

**Comment by flxzt** [comment]
I updated the poppler dependency and the issue seems to be fixed, at least with your pdf. If you want, you could try it out with a build from the github main branch (and report back), or wait until a new release

**Comment by alexpattyn** [comment]
Can confirm that this is fixed with the pdf attached above using v0.3.5. I'll report again if there are issues with other pdfs I come across. 

But I think this can be closed unless there were other issues.

**Comment by flxzt** [comment]
Okay! Good to hear
